### PR TITLE
Modify platform configuration of aarch64-qemu-virt-hv for GICv2 with virtualization

### PR DESCRIPTION
**Pull request by pengzechen**
GICv2 with Virtualization

### CODE REVIEW

File: platforms/aarch64-qemu-virt-hv.toml [hunk 1]
```diff
@@ -23,7 +23,7 @@ kernel-aspace-size = "0x0000_ffff_ffff_f000"
 # MMIO regions with format (`base_paddr`, `size`).
 mmio-regions = [
     ["0x0900_0000", "0x1000"],      # PL011 UART
-    ["0x0800_0000", "0x2_0000"],    # GICv2
+    ["0x0800_0000", "0x5_0000"],    # GICv2 with Virtualization
```
Comment (hky1999) [inline platforms/aarch64-qemu-virt-hv.toml]: Enrich the comment content.

For example: 
```toml
["0x0800_0000", "0x5_0000"], # GICv2 with Virtualization (GICV@0x0803_0000, GICH@0x0804_0000)
```

**Review by hky1999 — CHANGES_REQUESTED** [review]
Rename this PR to `Modify platform configuration of aarch64-qemu-virt-hv for GICv2 with virtualization`

**Comment by hky1999** [comment]
> Rename this PR to `Modify platform configuration of aarch64-qemu-virt-hv for GICv2 with virtualization`

As well as your commit msg, you do not support GICv2 with virtualization, you just did little modification in config file.

File: platforms/aarch64-qemu-virt-hv.toml (hunk 1 above)
Comment (pengzechen) [reply platforms/aarch64-qemu-virt-hv.toml]: Ok

**Review by hky1999 — COMMENTED** [review]
Remove this redundant commit 👉 58aa5b29b83b3a862db66299e1803bf53ae44fb5

**Review by hky1999 — APPROVED** [review]

**Pull request merged**